riotfile.py

```diff
@@ -49,8 +49,10 @@
         Venv(
             name="mypy",
             command="mypy {cmdargs}",
```

Comment:
```suggestion
            command="mypy --install-types --non-interactive {cmdargs}",
```

optional, but this could also work since it'll auto-install the `types-*` modules for you, and need `--non-interactive` to keep it from asking for confirmation to install.

Reply:
ah cool didn't realize you could have it install types while also type checking!

Reply:
done separately in #44